packages/protocol/contracts/L1/verifiers/SgxVerifier.sol

```diff
@@ -77,37 +81,29 @@ contract SgxVerifier is EssentialContract, IVerifier {
         ids = _addInstances(_instances);
     }
 
-    /// @notice Adds SGX instances to the registry by another SGX instance.
```

Comment:
So we won't support SGX instances verifying other SGX instances anymore? It's much cheaper, but of course does require additional code to do it in SGX so maybe not worth it anymore?

Depending on our trust in the onchain code, verifying the RA in a smart contract + SGX may also be worth it from a security perspective, but of course again only if we think it's worth it to support the SGX verification path as well.

Reply:
I think (or thought) this is what we agreed:
<img width="557" alt="kép" src="https://github.com/taikoxyz/taiko-mono/assets/51912515/08c75c00-b10b-4201-bf0b-6f8f570886c8">

Reply:
so no 'addInstances' by trusted instances.. they can/should add themselves.

Reply:
As discussed - leave it as is (for now) and when SGX is ready to support registration service, we can add it back.